tools/model_tools/src/openvino/model_zoo/quantize_engine/quantizer.py

```diff
@@ -92,10 +92,11 @@ def get_pot_cmd_prefix(self) -> List[str]:
         return pot_cmd_prefix
 
 
-    def quantize(self, reporter, model, precision, target_device, pot_cmd_prefix, pot_env) -> bool:
+    def quantize(self, reporter, model, precision, target_device, pot_cmd_prefix, pot_env, model_root=None) -> bool:
         input_precision = _common.KNOWN_QUANTIZED_PRECISIONS[precision]
 
-        pot_config_base_path = _common.MODEL_ROOT / model.subdirectory / 'quantization.yml'
+        model_root = _common.MODEL_ROOT if model_root is None else model_root
```

Comment:
is there any validation steps for model_root providing (e.g. that it is exists and it is directory) or we should not take care about it and it is handled in low-level components?